iceoryx_utils/test/moduletests/test_relocatable_ptr.cpp

```diff
@@ -75,8 +75,6 @@ TEST_F(RelocatablePointer_test, relocation)
     uint8_t* base1 = block1[0];
     uint8_t* base2 = block2[0];
 
-    EXPECT_EQ(base2 - base1, BLOCK_SIZE);
```

Comment:
why is this removed?

Reply:
I discussed it with @MatthiasKillat and this line is not needed for the test since it actually does not test any kind of behavior.